src/components/DocNode/DocNode.tsx

```diff
@@ -0,0 +1,141 @@
+
+import classes from './DocNode.module.scss';
+import indexClasses from 'styles/index.module.scss';
+
+interface DocNodeProps {
+  pathwayState: GuidanceState | State;
+  isActionable: boolean;
+  documentation: DocumentationResource | undefined;
+  isGuidance: boolean;
+}
+
+const DocNode: FC<DocNodeProps> = ({ pathwayState, isActionable, documentation, isGuidance }) => {
+  let fhirFields: ReactElement[] = [];
+  if (documentation && documentation.resource) {
+    fhirFields = parseResource(documentation.resource);
+  }
+  const guidance = isGuidance && renderRecGuidance(pathwayState as GuidanceState);
+
+  return (
+    <div className={indexClasses.docNode}>
+      <table className={classes.infoTable}>
+        <tbody>
+          {guidance}
+          {fhirFields}
+        </tbody>
+      </table>
+      {isActionable && (
+        <form className={classes.commentsForm}>
+          <label>Comments:</label>
+          <button className={indexClasses.button} onClick={(e): void => e.preventDefault()}>
+            Use Default Text
+          </button>
+          <textarea className={classes.comments}></textarea>
+          <button
+            className={`${classes.accept} ${indexClasses.button}`}
+            onClick={(e): void => e.preventDefault()}
+          >
+            Accept
+          </button>
+          <button
+            className={`${classes.decline} ${indexClasses.button}`}
+            onClick={(e): void => e.preventDefault()}
+          >
+            Decline
+          </button>
+        </form>
+      )}
+    </div>
+  );
+};
+
+type DocNodeFieldProps = {
+  title: string;
+  description: ReactNode;
+};
+
+const DocNodeField: FC<DocNodeFieldProps> = ({ title, description }) => {
+  return (
+    <tr>
+      <td className={classes.descTitle}>{title}</td>
+      <td className={classes.desc}>{description}</td>
+    </tr>
+  );
+};
+
+function renderRecGuidance(pathwayState: GuidanceState): ReactElement[] {
+  const resource = pathwayState.action[0].resource;
+  const coding =
+    'medicationCodeableConcept' in resource
+      ? resource.medicationCodeableConcept.coding
+      : resource.code.coding;
+  return [
+    <DocNodeField key="Notes" title="Notes" description={pathwayState.action[0].description} />,
+    <DocNodeField key="Type" title="Type" description={resource.resourceType} />,
+    <DocNodeField
+      key="System"
+      title="System"
+      description={
+        <>
+          {coding[0].system}
+          <a href={coding[0].system} target="_blank" rel="noopener noreferrer">
+            <FontAwesomeIcon icon="external-link-alt" className={classes.externalLink} />
+          </a>
+        </>
+      }
+    />,
+    <DocNodeField key="Code" title="Code" description={coding[0].code} />,
+    <DocNodeField key="Display" title="Display" description={coding[0].display} />
+  ];
+}
+function parseResource(resource: fhir.DomainResource): ReactElement[] {
```

Comment:
DocNode is starting to become responsible for a lot (with this function it has to know how to **parse and extract information from resources** AND **display them**).  Seems like we might want to think about breaking this up.

I think we need to think about this more to get the right solution, so I won't hold everything up on this, but it would be great from a reuse and maintainability perspective if we could break this up and out.